deploy-manage/deploy/cloud-on-k8s/deploy-eck-on-openshift.md

```diff
@@ -1,4 +1,6 @@
 ---
+applies:
+  eck: all
 mapped_pages:
   - https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-openshift.html
 ---
```

Comment:
since this is a series of child pages, it should be updated to say "This section"